src/core/utils/helpers.ts

```diff
@@ -135,3 +135,16 @@ export const dateMatchesUsFormat = (date: string | null) => {
   const returnValue = dateFound && dateFound.length > 0 ? dateFound[0] : null;
   return returnValue;
 };
+
+export const usDateStringToEurDateObj = (date: string | null | undefined) => {
+  if (!date || date === null) {
+    return date;
+  }
+  const dateArray = date?.split("-");
+  const euDate = new Date(
+    Number(dateArray[0]),
+    Number(dateArray[1]) - 1,
+    Number(dateArray[2])
+  );
+  return euDate;
```

Comment:
Consider if this can be replaced by

```js
var customParseFormat = require('dayjs/plugin/customParseFormat');
dayjs.extend(customParseFormat);
return dayjs(date, 'YYYY-MM-DD');
```
https://day.js.org/docs/en/plugin/custom-parse-format

Reply:
Perf! Thanks for the suggestion 💯